ha/core/controllers/pcs/cluster_controller.py

```diff
@@ -38,7 +48,37 @@ def start(self) -> dict:
             ([dict]): Return dictionary. {"status": "", "msg":""}
                 status: Succeeded, Failed, InProgress
         """
-        raise HAUnimplemented("Cluster start operation is not implemented.")
+        _, _, _rc = self._execute.run_cmd(const.PCS_CLUSTER_STATUS, check_error=False)
+        if _rc != 0:
+            self._execute.run_cmd(const.PCS_CLUSTER_START, check_error=False)
+            time.sleep(60)
+            _, _, _rc = self._execute.run_cmd(const.PCS_CLUSTER_STATUS, check_error=False)
+            if _rc != 0:
+                return {"status": const.STATUSES.FAILED.value, "msg": "Cluster start operation failed"}
+
+        _res = self.node_list()
+        _res = json.loads(_res)
```

Comment:
`_res`

add nodelist output in log info

Reply:
Added log to print node list when _res returns success